### PR TITLE
TsFile Docs Update for the hierarchy of TsFile

### PR DESCRIPTION
**Pull request by mikip65**

### CODE REVIEW

File: docs/Documentation/UserGuideV0.7.0/7-TsFile/3-Hierarchy.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+-->
+#TsFile Hierarchy
+  Here is a brief introduction of the structure of a TsFile file.
+  
+##Variable Storage
+ * **Big Endian**
+        
+     * For Example, the `int` `0x8` will be stored as `00 00 00 08`, not `08 00 00 00`
+  
+ * **String with Variable Length**
```
Comment (jixuan1989) [inline docs/Documentation/UserGuideV0.7.0/7-TsFile/3-Hierarchy.md]: like this part very much!

**Comment by samperson1997** [comment]
Sorry that I had deleted "3-Hierarchy" in the file `0-content.md` in [PR #265](https://github.com/apache/incubator-iotdb/pull/265) to ensure the website link won't have bugs for now. Please update `0-content.md` and add "3-Hierarchy" in Chapter 7 again :-)

**Comment by mikip65** [comment]
TsFileBreakdown

**Comment by mikip65** [comment]
TsFileBreakdown_Edited

**Comment by jixuan1989** [comment]
I will merge this PR. the failed Travis is caused by a known reason and has been fixed in #282

**Comment by jixuan1989** [comment]
Hi, 
I think you have to resubmit the PR because this PR contains some commits which is in the master branch while not in the rel/0.8...  @mikip65 

Otherwise we can merge this PR into the master branch. @qiaojialin

**Comment by qiaojialin** [comment]
> Hi,
> I think you have to resubmit the PR because this PR contains some commits which is in the master branch while not in the rel/0.8... @mikip65
> 
> Otherwise we can merge this PR into the master branch. @qiaojialin

resubmit is good

**Pull request closed without merging**